openedx/core/djangoapps/user_authn/views/password_reset.py

```diff
@@ -641,9 +642,18 @@ def password_change_request_handler(request):
     else:
         return HttpResponseBadRequest(_("No email address provided."))
 
+class ResetTokenValidationThrottle(AnonRateThrottle):
+    """
+    Setting rate limit for token validation
+    """
+    rate = settings.RESET_PASSWORD_TOKEN_VALIDATE_API_RATELIMIT
```

Comment:
`30/7d` format won't work with DRF throttle, you need to use `ratelimit` package for this.
https://github.com/edx/edx-platform/blob/bcfe2874fc3816a8e87fc13286cf51e1e0961fcc/openedx/core/djangoapps/user_authn/views/register.py#L729-L731